Mathlib/Algebra/GroupWithZero/Action/Opposite.lean

```diff
@@ -63,3 +62,4 @@ open MulOpposite
 instance CancelMonoidWithZero.toFaithfulSMul_opposite [CancelMonoidWithZero α]
     [Nontrivial α] : FaithfulSMul αᵐᵒᵖ α :=
   ⟨fun h => unop_injective <| mul_left_cancel₀ one_ne_zero (h 1)⟩
+#align cancel_monoid_with_zero.to_has_faithful_opposite_scalar CancelMonoidWithZero.toFaithfulSMul_opposite
```

Comment:
Uh oh.

Reply:
Tried very hard to not reintroduce them... 😓